docs/Setup.md

```diff
@@ -24,8 +24,18 @@ This will walk you through installing HuskSync on your network of Spigot servers
 <summary><b>For MongoDB Users</b></summary>
```

Comment:
Could we mention postgres in Step 3? i.e.
```markdown
### 3. Enter Database & Redis server credentials
- Navigate to the HuskSync config file on each server (`~/plugins/HuskSync/config.yml`)
- Under `credentials` in the `database` section, enter the credentials of your database (MySQL/MariaDB/MongoDB/PostgreSQL). You shouldn't typically need to modify the `connection_pool` properties.
```

Reply:
done 👍